app/assets/stylesheets/decidim/modules/_navbar.scss

```diff
@@ -156,6 +152,22 @@ $menuitem-padding: 1em 2em;
   }
 }
 
+.nav-row {
```

Comment:
ℹ️ I think the city of helsinki site uses blueprint. There's some sort of row and column system going on, and to stretch the left and right parts of the header outside the max-width main body content, I had to roll my own version of .row that had different padding rules. Maybe there was another way to make this work within the system rather than just hacking some styles in?